### ISSUE TITLE
Solium lints incorrectly on windows.

### ISSUE THREAD

**Issue by dev-matan-tsuberi** [duaraghav8/Solium#162]
Running `yarn solium` on windows 10 and on a docker container with Ubuntu installed yields very different results.

Reproduce: 
`$ git clone https://github.com/daostack/daostack.git`
`$ cd daostack`
`$ yarn`
`$ yarn solium`

On windows: 47 errors, 205 warnings found.
On Ubuntu: only 5 warns

**Comment by duaraghav8** [comment]
Could you please produce the following for both win & ubuntu:

- Command used to lint your file(s) (or whether you're using solium as an IDE extension)
- `.soliumrc.json` & `.soliumignore`
- Solium version (`solium -V`)
- screenshot of outputs

**Comment by dev-matan-tsuberi** [comment]
* No IDE extension, same exact command on both `yarn solium`

`soliumrc.json`:
```
{
  "extends": "solium:all",
  "plugins": [
    "security"
  ],
  "rules": {
    "quotes": [
      "error",
      "double"
    ],
    "indentation": [
      "error",
      4
    ],
    "arg-overflow": [
      "warning",
      5
    ]
  }
}
```

`.soliumignore:`:
```
node_modules
docs
```

* `yarn solium -V` - `1.1.0`
* screenshots:

windows:
      
![image](https://user-images.githubusercontent.com/33967618/34570768-6b47a728-f175-11e7-9bd2-69dc22373bcc.png)

inside docker node (`docker run -it node /bin/bash`):
![image](https://user-images.githubusercontent.com/33967618/34570905-d15d1a98-f175-11e7-9601-4557c4359fd1.png)

**Comment by duaraghav8** [comment]
@dev-matan-tsuberi **The output you see on Ubuntu is the correct one.**

## Problem
You must've noticed that all (false positive) issues in windows stem from the whitespace-related rules - `blank-lines`, `indentation`, `whitespace` - they all analyse linebreaks in your code. Line break characters are usually different in un Unix (`\n`) and Windows (`\r\n`) and I should've accounted for the latter too.

^I'm yet to confirm this empirically but pretty sure this is the reason.

This went undetected for the past 1.5 years probably because almost all devs are using solium on Unix. Since I've always run Solium on Linux & Mac, I never noticed this myself!

Thanks for reporting this (massive) bug
Bottom line: **Please use Solium on linux or Mac while I work on a fix**:)

**Comment by GabrielAlacchi** [comment]
@duaraghav8 PR #161 should fix the blank-lines rule for windows, since I changed the regular expressions to account for the `\r\n` line endings where necessary.

**Comment by dev-matan-tsuberi** [comment]
Is this being fixed?

**Comment by duaraghav8** [comment]
@dev-matan-tsuberi It has been partially fixed in the latest release (blank-lines shouldn't give you trouble any more) and will be completely fixed in the upcoming release (4-5 days)

**Comment by duaraghav8** [comment]
v1.1.5 fully resolves this issues. @dev-matan-tsuberi I no longer get any unexpected issues on my windows machine.
Please let me know if the issue still persists for you.
